cmd/config.go

```diff
@@ -259,3 +260,29 @@ type PortConfig struct {
 	HTTPSPort int
 	TLSPort   int
 }
+
+// BeelineConfig provides config options for the Honeycomb beeline-go library,
+// which are passed to its beeline.Init() method.
+type BeelineConfig struct {
+	// WriteKey is the API key needed to send data Honeycomb. Never check in a
+	// value for this config parameter in a publicly-readable repository.
+	WriteKey string
+	// Dataset is the event collection, e.g. Staging or Prod.
+	Dataset string
+	// ServiceName is the name of this service, e.g. WFE or RA.
+	ServiceName string
+	// Mute disables honeycomb entirely; useful in test environments.
+	Mute bool
```

Comment:
Why "mute" rather than omitting the whole beeline config?

Reply:
Because it allows our test configs to be as close as possible to the real configs -- just a `sed -i -e s/"mute": true/"writeKey": "deadbeef"/` away from being the real thing. I find that much less confusing than "why do our prod configs have this entire stanza that our test configs leave out?". It also leaves us room for things like having the beeline library log to stdout during tests if we want, eventually.